ZenPacks/zenoss/Import4/service_definition/imp4mariadb.json

```diff
@@ -0,0 +1,171 @@
+{
+    "servicePath": "/",
+    "serviceDefinition" : {
+        "Name": "Imp4MariaDB",
+        "Description": "MariaDB (mysql) environment for import4",
+        "Launch": "manual",
+        "Command": "/usr/bin/sleep 1000000",
```

Comment:
At this stage, the whole service def is temp until refined to contain env needed for the import scripts. It is also possible that we actually start a daemon to handle all the GUI requests. TBD.